stronk/models/exercise.py

```diff
@@ -1,10 +1,37 @@
+from psycopg2.errors import UniqueViolation
+from sqlalchemy.exc import DBAPIError, IntegrityError
+from werkzeug.exceptions import BadRequest, Conflict, InternalServerError, NotFound
+
 from stronk import db
 
+
 class Exercise(db.Model):
     id = db.Column(db.Integer, primary_key=True)
     name = db.Column(db.String(128), index=True, nullable=False, unique=True)
     description = db.Column(db.String(500), nullable=False)
 
+    @staticmethod
+    def create(name, description):
+        exercise = Exercise(name=name, description=description)
+
+        try:
+            db.session.add(exercise)
+            db.session.commit()
+
+            return exercise
+        except IntegrityError as err:
+            raise BadRequest("Name already used by another exercise")
```

Comment:
Should we consider some sort of normalization for the exercise name? Ie. `Richard's exercise` and `richard's exercise` are currently treated as separate names, but they should be the same

Reply:
@13ABEL @MrRDotS how do we feel about just capitalizing each word in the workout name then? For instance, "Richard's Exercise".

Reply:
yeee that's good for me